configs/test.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Its a good idea to think about whether we want to have multiple config files or a single config file with all configs of all environments

Reply:
I'm thinking of making another repo - it would contain: 
1) The docker compose files
2) The config files
Then pisa would stay as pisa, and this other repo would be a kind of 'deployment' repo. What dyou think?

A different way would be to create a *package* step. This would build the docker image, publish it, then create a package that contains the docker compose files and an example config.

Reply:
I prefer the package creation instead of another repository, the easiest way to do this is to save the packages in circleci artifacts and a link can be shared for anyone willing to download

Reply:
OK, agreed. I'll try to organise the configs for this.